docs/_includes/common/filter-menu.html

```diff
@@ -0,0 +1,472 @@
+                    </div>
+                </div>
+            </span>
+        </div>
+        {% highlight html %}
+<span class="filter-menu">
+    <div class="filter-menu__items" role="menu">
+        <div class="filter-menu__scroll-area">
+            <div class="filter-menu__item" role="menuitem">
+                <span class="checkbox">
+                    <input aria-label="filter menu checkbox example" class="checkbox__control" type="checkbox" name="filter-menu-checkbox-item-1" tabindex="-1" />
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 1 with a long string that will wrap to another line</span>
+            </div>
+            <div class="filter-menu__item" role="menuitem">
+                <span class="checkbox">
+                    <input aria-label="filter menu checkbox example" class="checkbox__control" type="checkbox" name="filter-menu-checkbox-item-2" tabindex="-1" />
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 2</span>
+            </div>
+            <div class="filter-menu__item" role="menuitem">
+                <span class="checkbox">
+                    <input aria-label="filter menu checkbox example" class="checkbox__control" type="checkbox" name="filter-menu-checkbox-item-3" tabindex="-1" />
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 3</span>
+            </div>
+        </div>
+        <div class="filter-menu__item filter-menu__item--footer" role="menuitem">
```

Comment:
The footer will have to be a button so it gets focus, right?  And does it get the same background focus style as the other items?

Reply:
It does not have to be a button. It just has to have `role="menuitem"`. Yes, it has the same background focus as the other menu items.

Reply:
In the form version it must be a button with type=submit. For the ARIA version it can be either role=menuitem or a button. Right now I don't have a strong preference for either. The markup restrictions for a11y semantics may dictate which we go with...